contracts/standard/proxy/ArbitratorVersioningProxy.sol

```diff
@@ -9,26 +10,30 @@ import "./VersioningProxy.sol";
  *  @author Enrique Piqueras - <epiquerass@gmail.com>
  *  @notice An Arbitrator proxy that only exposes methods in the Arbitrator spec.
  */
-contract ArbitratorVersioningProxy is Arbitrator, VersioningProxy {
+contract ArbitratorVersioningProxy is Arbitrator, Arbitrable, VersioningProxy {
     /* Structs */
 
-    struct Dispute {
+    // Warning! Overrides DisputeStruct from parent.
+    struct DisputeStruct {
+        Arbitrable arbitrated;
+        uint externalDisputeID;
         Arbitrator arbitrator;
-        uint256 disputeID;
         uint256 choices;
```

Comment:
This happens more than once throughout the contract.

Let's just use `uint` for consistency.